### PR TITLE
X11, intltool @ GCCcore

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Immediately after submitting this, I noticed #7872 
I'll change it.

**Comment by Micket** [comment]
I also totally missed that inject-checksums added duplicated checksums. Dang. Again.  I'll update this.

**Comment by Micket** [comment]
I thought printproto was part of xorgproto, but I got hit with
```
checking for XPRINT... no
configure: error: Package requirements (x11 >= 1.6 xext xextproto xau printproto) were not met:

No package 'printproto' found
```

File: easybuild/easyconfigs/x/X11/X11-20190313-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,212 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20190313'
```
Comment (boegel) [inline easybuild/easyconfigs/x/X11/X11-20190313-GCCcore-8.2.0.eb]: @Micket Traditionally, we've used the date of the most recent component as version for the bundle, so this should be `20190311`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/efa795e4ac8400e0c814affcce780383 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7783482a632c3a89c11a8e821e1ae906 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2012.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3f20da1f847edd3c429090cc72eec3e9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

File: easybuild/easyconfigs/x/X11/X11-20190311-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('fontconfig', '2.13.1'),
+    ('freetype', '2.9.1'),
+    ('zlib', '1.2.11'),
+]
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('Autotools', '20180311'),
+    ('Bison', '3.0.5'),
```
Comment (ocaisa) [inline easybuild/easyconfigs/x/X11/X11-20190311-GCCcore-8.2.0.eb]: Why is this old version of Bison required?

Comment (ocaisa) [reply easybuild/easyconfigs/x/X11/X11-20190311-GCCcore-8.2.0.eb]: Sorry, my bad, didn't notice the toolchain

Comment (boegel) [reply easybuild/easyconfigs/x/X11/X11-20190311-GCCcore-8.2.0.eb]: It's not required, it's good enough, and we don't have easyconfigs for newer `Bison` versions yet that use `GCCcore/8.2.0`

**Pull request merged**